scipy/stats/tests/test_distributions.py

```diff
@@ -4154,6 +4162,14 @@ def test_location_scale(
     ):
         """Tests for pdf and cdf where loc, scale are different from 0, 1
         """
+
+        uname = platform.uname()
+        is_linux_32 = uname.system == 'Linux' and "32bit" in platform.architecture()[0]
+        # Test seems to be unstable (see gh-17839 for a bug report on Debian
+        # i386), so skip it.
+        if is_linux_32 and case == 'pdf':
+            pytest.skip("%s fit known to fail or deprecated" % dist)
```

Comment:
```suggestion
            pytest.skip("Test unstable on some platforms; see gh-17839, 17859")
```
https://github.com/scipy/scipy/pull/17865/commits/28fb42e20124010b3837d3fe1c90f5ac89f34414#r1088471394